extensions/skins/conversation_v1/player.html

```diff
@@ -201,6 +201,32 @@
     </div>
   </div>
 
+  <div ng-if="isOnTerminalCard() && !isIframed && isCurrentCardAtEndOfTranscript() && (recommendedExplorationSummaries.length > 0 || (isInEditorPage && invalidExpIds.length > 0))"
```

Comment:
I think the `(isInEditorPage && invalidExpIds.length > 0))` clause is no longer relevant because the warning is displayed in the end interaction?